src/main/java/org/synyx/urlaubsverwaltung/person/basedata/PersonBasedataEntity.java

```diff
@@ -0,0 +1,60 @@
+package org.synyx.urlaubsverwaltung.person.basedata;
+
+import org.synyx.urlaubsverwaltung.person.Person;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToOne;
+import javax.persistence.PrimaryKeyJoinColumn;
+import javax.persistence.Table;
+
+import static javax.persistence.FetchType.LAZY;
+
+@Entity
+@Table(name = "person_basedata")
+class PersonBasedataEntity {
+
+    @Id
+    @Column(name = "person_id")
+    private Integer personId;
+
+    @OneToOne(fetch = LAZY)
```

Comment:
why do we fetch it lazy? Do we not need it in every case?

Reply:
Currently, its uses in every case, removed lazy fetching